packages/nc-gui/components/cell/RichText.vue

```diff
@@ -156,11 +156,12 @@ watch(editorDom, () => {
 
 <template>
   <div
-    class="h-full"
+    class="h-full focus:!outline-none focus-visible:!outline-none"
```

Comment:
what to override?

![image](https://github.com/nocodb/nocodb/assets/35857179/8bcc872f-80d4-4bf8-800d-3b80fb33fa23)